package.json

```diff
@@ -40,7 +40,7 @@
     ]
   },
   "scripts": {
-    "postinstall": "patch-package",
+    "postinstall": "patch-package --patch-dir patches0 && patch-package",
```

Comment:
As suggested in https://github.com/ds300/patch-package/issues/203#issuecomment-616187572